### DISCUSSION TITLE
How can I sort string column?

### DISCUSSION THREAD

**Discussion by Jufien1** [prisma/prisma#12764] [Q&A] [answered]
I use MySQL.

A column called 'AGE' contains numbers in the form of a string.

If I use prisma to sort, it's not 1-2-...-9-11-88-333

1-11-2-333-...-88-9 Sort like this.

I would like to know if there is any way to sort other than using $queryRaw.

**Comment by nurul3101** [comment, marked as answer]
Hey @Jufien1 👋 

You can sort the records after you get the response from the prisma query.
You could use `parseInt` to convert strings to Integer and then can use any sorting method E.g. [sortBy](https://lodash.com/docs/4.17.15#sortBy)

Another option could be to cast your string values to Integer before using orderBy but this would include the use of the `queryRaw` operation.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
